modules/nw-exposing-router-metrics.adoc

```diff
@@ -0,0 +1,149 @@
+// Module included in the following assemblies:
+//
+// * networking/ingress_operator.adoc
+
+:_content-type: PROCEDURE
+[id="nw-exposing-router-metrics_{context}"]
+= Exposing router metrics
+
+You can expose the HAProxy router metrics by default in Prometheus format on the default stats port, 1936. The external metrics collection and aggregation systems such as Prometheus and statsd can access the HAProxy router metrics. You can view the HAProxy router metrics in a browser in the HTML and cluster service version (CSV) format.
+
+.Prerequisites
+
+* You configured your firewall to access the default stats port, 1936.
+
+.Procedure
+
+. Create a router by running the following command:
++
+[source,terminal]
+----
+$ oc adm router --service-account=router
+----
```

Comment:
We don't need to create router in OCP v4.x since it is already installed by default, and `oc adm router` command is not supported in 4.x.

Reply:
@lihongan @ShudiLi Addressed.